tasks/skylab/HISAT2.wdl

```diff
@@ -9,6 +9,9 @@ task HISAT2PairedEnd {
     String output_basename
     String input_id
 
+    String? FQ1=""
```

Comment:
Since these are defined within the bash and used in the dame command, you should not need these optional input (here of in the singleEnd version)

Reply:
Done.